Sources/Utils/Extensions/AVAsset+Extensions.swift

```diff
@@ -54,18 +54,18 @@ extension AVAsset {
   // MARK: - Description
 
   var g_videoDescription: CMFormatDescription? {
-    if let object = tracks(withMediaType: AVMediaTypeVideo).first?.formatDescriptions.first {
-      return object as! CMFormatDescription
+    guard let object = tracks(withMediaType: AVMediaTypeVideo).first?.formatDescriptions.first else {
+      return nil
     }
 
-    return nil
+    return (object as! CMFormatDescription)
   }
 
   var g_audioDescription: CMFormatDescription? {
-    if let object = tracks(withMediaType: AVMediaTypeAudio).first?.formatDescriptions.first {
-      return object as! CMFormatDescription
+    guard let object = tracks(withMediaType: AVMediaTypeAudio).first?.formatDescriptions.first else {
+      return nil
     }
 
-    return nil
+    return (object as! CMFormatDescription)
```

Comment:
I think you can move the casting into the `guard` as well to get rid of the force unwrapping :)

Reply:
swift compiler gives an error - "Conditional downcast to CoreFoundation type 'CMFormatDescription' will always succeed". So 'as?' in guard statement can not be done :)

Reply:
Aha, that is interesting :)